### ISSUE TITLE
[flaky ci] [Serial]SRIOV VirtualMachineInstance with sriov plugin interface [test_id:1754]should create a virtual machine with sriov interface

### ISSUE THREAD

**Issue by dhiller** [kubevirt/kubevirt#4519]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

/triage build-watcher

**What happened**:

Test `[Serial]SRIOV VirtualMachineInstance with sriov plugin interface [test_id:1754]should create a virtual machine with sriov interface` flaked before merge of #4501 

PR history: https://prow.apps.ovirt.org/pr-history/?org=kubevirt&repo=kubevirt&pr=4501

Offending run: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4511/pull-kubevirt-e2e-kind-1.17-sriov/1325727568359329793

Error stacktrace:

```
tests/vmi_multus_test.go:697
Unexpected error:
    <*errors.StatusError | 0xc002db5f40>: {
        ErrStatus: {
            TypeMeta: {Kind: "", APIVersion: ""},
            ListMeta: {
                SelfLink: "",
                ResourceVersion: "",
                Continue: "",
                RemainingItemCount: nil,
            },
            Status: "Failure",
            Message: "etcdserver: request timed out",
            Reason: "",
            Details: nil,
            Code: 500,
        },
    }
    etcdserver: request timed out
occurred
GOROOT/src/reflect/value.go:460
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by dhiller** [comment]
Possibly related:
* https://github.com/kubevirt/kubevirt/issues/3777
* https://github.com/kubevirt/kubevirt/issues/3742

**Comment by oshoval** [comment]
Suggestion (if the network team and reviewers would like)
Try empirically this one, as i saw several times that a freshly created cluster might suffer in the beginning from etcd timeouts,
even that cluster-up is "done"
https://github.com/kubevirt/kubevirtci/pull/473

More possible refs in the PR

**Comment by oshoval** [comment]
Another run 
https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4501/pull-kubevirt-e2e-kind-1.17-sriov/1325742799731036160

**Comment by oshoval** [comment]
https://github.com/kubernetes/kubernetes/issues/70082#issuecomment-433604939

looks interesting, checking the metrics of the disk IO
this issue is alive 7 days ago, and lots of people reported it
we do see the prints mentioned in the issue (will add link to the PR 473 thread with the prints once github will stabilize)

**Comment by ormergi** [comment]
I can confirm we do get `..took to long to execute` warnings all over etcd pods logs on every sriov lane job, here are few for example
Right after sriov-operator deployment
[1326381124888301569](https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4512/pull-kubevirt-e2e-kind-1.17-sriov/1326381124888301569) [etcd logs](https://storage.googleapis.com/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4512/pull-kubevirt-e2e-kind-1.17-sriov/1326381124888301569/artifacts/pods/0_kube-system_etcd-sriov-control-plane-etcd.log)
Right after kubevirt-operator deployment, when this error occurs at this time it will affect the first test 
[1326105060454699008](https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4435/pull-kubevirt-e2e-kind-1.17-sriov/1326105060454699008) [etcd logs](https://storage.googleapis.com/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4435/pull-kubevirt-e2e-kind-1.17-sriov/1326105060454699008/artifacts/k8s-reporter/pods/1_kube-system_etcd-sriov-control-plane-etcd.log)

We followed this issue https://github.com/kubernetes-sigs/kind/issues/717, where its suggests to manipulate inotify watcher configurations in order to fix etcd timeout errors.
I verified and on all the CI nodes that schedule sriov lane jobs we configured:
```# cat /etc/sysctl.d/fs.inotify.max_user_watches.conf 
fs.inotify.max_user_watches=524288
fs.inotify.max_user_instances=8192
vm.min_free_kbytes=3000000```
On those nodes we use SAS disks so it could be low disk performance, 
personally I would expect etcd to work properly on hard drives as well.

**Comment by ormergi** [comment]
/close 

Should be fixed by #4588
